addons/mail/models/mail_blacklist.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class MailBlackList(models.Model):
+    """ Model of blacklisted email addresses to stop sending emails."""
+    _name = 'mail.blacklist'
+    _inherit = ['mail.thread']
+    _description = 'Mail Blacklist'
+    _rec_name = 'email'
+
+    email = fields.Char(string='Email Address', required=True, index=True, help='This field is case insensitive.',
+                        track_visibility='always')
+    active = fields.Boolean(default=True, track_visibility='onchange')
+
+    _sql_constraints = [
+        ('unique_email', 'unique (email)', 'Email address already exists!')
+    ]
+
+    @api.model
+    def create(self, values):
+        """ To avoid crash during import due to unique email, return the existing record if any """
+        rec = self.search([('email', '=ilike', values['email'])])
```

Comment:
Does it work with the `create_multi`?
I think, since your method is decorated with `api.model`, it will execute a search by create. It could be interesting to decorated it with `@api.model_create_multi` and optimize your search to work in batch.